packages/core/src/text/Display.tsx

```diff
@@ -0,0 +1,51 @@
+import { makePrefixer } from "@jpmorganchase/uitk-core";
```

Comment:
```suggestion
import { makePrefixer } from "../utils";
```

Reply:
Thank you! It was in front of my eyes, I just couldn't see it